lib/open_project/access_control.rb

```diff
@@ -176,6 +176,7 @@ def contract_actions_map
       def grant_to_admin?(permission_name)
         # Parts of the application currently rely on granting not defined permissions,
         # e.g. :edit_attribute_help_texts to administrators.
+        # TODO: When we have added all permissions, we should remove this check.
```

Comment:
The method is also no longer used in the `UserPermissibleService`. When we turn off the `UserALlowedService`, we can get rid of this as well